src/views/_screen-sm-max.scss

```diff
@@ -61,3 +61,56 @@ h2,
 		}
 	}
 }
+
+/* Responsive tables cards */
+.provisional {
+	&.gc-table {
+		.left-xxs {
+			clear: both;
+			display: block;
+			text-align: left;
+		}
+		>tbody {
+			>tr {
+				>td {
+					&:first-child {
+						border-top: none;
+				}
+			}
+		}
+	}
+		tr {
+			border: 1px solid #ddd;
+			display: block;
+			margin-bottom: .625em;
+			padding: .35em;
+		}
+		border: 0;
```

Comment:
Should this be here, in the middle of children selectors? Declared here will apply the rule on `.gc-table` so it could be at line 68.

Reply:
Doesn't seem to actually do anything, despite my notes of what it's for, I've removed it.